src/css/custom.css

```
@@ -151,16 +151,43 @@ h5 {
     font-size: 18px;
 }
 
+/* Definition/description lists styling with flex*/
+
+dl {
+  display: flex;
+  flex-flow: row wrap;
+}
 dt {
-    font-weight:700;
+  flex-basis: 20%;
+  padding: 2px 8px;
+  text-align: left;
+  font-weight: 700;
+}
+dd {
```

```suggestion
dd .noun-definition {
```

Try something like `.noun-definition`  then apply the class